calc/src/App.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Buttons from "./Components/Buttons";
+import "./App.css";
+
+class App extends React.Component {
+  state = {
+    inputA: "0",
+    inputB: "0",
+    operator: "",
+    complete: false,
+  };
+
+  numberButton = (value) => {
+    if (this.state.inputA === "0" || this.state.complete === true) {
+      this.setState({ inputA: value, complete: false });
+    } else {
+      this.setState((prevState) => ({
+        inputA: prevState.inputA + value,
+      }));
+    }
+  };
+
+  operators = (value) => {
+    const arr = ["+", "-", "*", "÷"];
+    for (let i = 0; i < arr.length; i++) {
+      if (value === arr[i]) {
+        this.setState((prevState) => ({ inputB: prevState.inputA }));
+        this.setState({ operator: arr[i] });
+        this.setState({ inputA: "0" });
+      }
```

Comment:
I'm a bit confused by this code and why we need to do any loop. The value would already be the operator, and this function should only be called with operators, therefore there's no need to check. 
this.setState((prevState) => ({inputB: prevState.inputA, operator: value, inputA: "0"}))